src/test/java/org/cyclops/integrateddynamics/core/evaluate/variable/TestParseOperators.java

```diff
@@ -0,0 +1,605 @@
+package org.cyclops.integrateddynamics.core.evaluate.variable;
+
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.nbt.NBTTagList;
+import org.cyclops.integrateddynamics.api.evaluate.EvaluationException;
+import org.cyclops.integrateddynamics.api.evaluate.variable.IValue;
+import org.cyclops.integrateddynamics.api.evaluate.variable.IVariable;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.fail;
+
+import static org.cyclops.integrateddynamics.core.evaluate.operator.Operators.*;
+
+/**
+ * Test the different parse operators.
+ * @author LostOfThought
+ */
+public class TestParseOperators {
+
+    private static IVariable[] s(String v){
+        IVariable[] ret = new IVariable[1];
+        ret[0] = new DummyVariableString(ValueTypeString.ValueString.of(v));
+        return ret;
+    }
+    
+    @Before
+    public void before() {
+
+    }
+
+    /**
+     * ----------------------------------- INTEGER -----------------------------------
+     */
+    @Test
+    public void testParseInt_IsInt() throws EvaluationException {
+        IValue res1 = PARSE_INTEGER.evaluate(s("0"));
+        assertThat("parse_Integer(\"0\") is an integer", res1, instanceOf(ValueTypeInteger.ValueInteger.class));
+    }
+    @Test
+    public void testParseIntEmpty() throws EvaluationException {
+        try {
+            PARSE_INTEGER.evaluate(s(""));
+            fail("parse_Integer(\"\") did not throw");
+        } catch (EvaluationException e){
+            assertThat("parse_Integer(\"\") throws", true, instanceOf(Boolean.class));
```

Comment:
I recommend using `@Test(expected = EvaluationException.class)` for these tests.